src/insns/scss_32bit.adoc

```diff
@@ -25,8 +25,8 @@ Description::
 `rd` is set to 1 if the tag of capabilities `cs1` and `cs2` are equal and the
 bounds and permissions of `cs2` are a subset of those of `cs1`.
 
-Capabilities with <<section_cap_malformed,malformed bounds>> are considered to
-have a base and top of 0.
+NOTE: If either `cs1` 's or `cs2` 's bounds are <<section_cap_malformed,malformed>>
+then the base and top bounds decode as zero, which causes this instruction to return zero.
```

Comment:
Again, not quite the same thing. If the bounds just decoded as zero, then we should be happy to say that a malformed cap is a subset of another malformed cap.

Reply:
true